File: ejercicios-py/ejercicio2.py
import random

def check_even_odd():
    number = random.randint(1, 100)
    print(f"El número generado es: {number}")

    if number % 2 == 0:
        print("El número es par.")
    else:
        print("El número es impar.")

check_even_odd()

File: ejercicios-js/ejercicio2.js
console.log ("ejercicio 2: verificador de par/impar")
// Generar un número aleatorio entre 1 y 100
const numero = Math.floor(Math.random() * 100) + 1;

// Verificar si es par o impar
const resultado = (numero % 2 === 0) ? "par" : "impar";

// Mostrar el resultado
('El número generado es ${numero} y es ${resultado}.') ;